390/Lauren_Low.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "census visualization"
+author: "Lauren Low"
+date: "September 16, 2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(dplyr)
+library(readr)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+```
+
+```{r loading csv and creating census table}
+urlfile1 = "https://raw.githubusercontent.com/390-2020-09-Fall/SCBI-ForestGEO-Data/master/tree_main_census/data/census-csv-files/scbi.stem1.csv"
+urlfile2 = "https://raw.githubusercontent.com/390-2020-09-Fall/SCBI-ForestGEO-Data/master/tree_main_census/data/census-csv-files/scbi.stem2.csv"
+urlfile3 = "https://raw.githubusercontent.com/390-2020-09-Fall/SCBI-ForestGEO-Data/master/tree_main_census/data/census-csv-files/scbi.stem3.csv"
+
+stem1 <- read_csv(url(urlfile1))
+stem2 <- read_csv(url(urlfile2))
+# stem3 <- read_csv(url(urlfile3))
+
+stems <- rbind(stem1, stem2)
+```
+
+```{r cleaning up/wrangling tree census data}
+ stems <- stems %>%
+  na.omit(stems) %>%
+  mutate(as.numeric(dbh))
+```
+
+```{r plotting avg_dbh over time}
+ggplot(data = stems, aes(dbh)) +
```

Comment:
Let's put some more informative labels, in particular for the facets. ideally your visualizations should be as "stand alone" as possible. For example, why not put the census year numbers?